package.json

```diff
@@ -3,9 +3,26 @@
     "version": "2.4.0",
     "description": "Svelte plugin for prettier",
     "main": "plugin.js",
+    "browser": "esm/standalone.mjs",
+    "exports": {
+        ".": {
+            "browser": {
+                "import": "./esm/standalone.mjs",
+                "default": "./standalone.js"
+            },
+            "default": "./plugin.js"
+        },
+        "./standalone": {
+            "import": "./esm/standalone.mjs",
+            "default": "./standalone.js"
+        },
+        "./package.json": "./package.json"
```

Comment:
I think we updated `rollup-plugin-svelte`, so there's no reason to export this anymore